app/translations/cy/LC_MESSAGES/messages.po

```diff
@@ -684,120 +686,129 @@ msgstr "Chwilio"
+#: app/templates/support_centre_list_of_centres.html:165
 msgid "Closed"
-msgstr ""
+msgstr "Ar gau"
 
-#: app/templates/support_centre_list_of_centres.html:67
+#: app/templates/support_centre_list_of_centres.html:69
 msgid "Tuesday"
-msgstr ""
+msgstr "Dydd Mawrth"
 
-#: app/templates/support_centre_list_of_centres.html:76
+#: app/templates/support_centre_list_of_centres.html:78
 msgid "Wednesday"
-msgstr ""
+msgstr "Dydd Mercher"
 
-#: app/templates/support_centre_list_of_centres.html:85
+#: app/templates/support_centre_list_of_centres.html:87
 msgid "Thursday"
-msgstr ""
+msgstr "Dydd Iau"
 
-#: app/templates/support_centre_list_of_centres.html:94
+#: app/templates/support_centre_list_of_centres.html:96
 msgid "Friday"
-msgstr ""
+msgstr "Dydd Gwener"
 
-#: app/templates/support_centre_list_of_centres.html:103
+#: app/templates/support_centre_list_of_centres.html:105
 msgid "Saturday"
-msgstr ""
+msgstr "Dydd Sadwrn"
 
-#: app/templates/support_centre_list_of_centres.html:112
+#: app/templates/support_centre_list_of_centres.html:114
 msgid "Sunday"
-msgstr ""
+msgstr "Dydd Sul"
 
-#: app/templates/support_centre_list_of_centres.html:121
+#: app/templates/support_centre_list_of_centres.html:123
 msgid "Census Saturday, 20 March"
-msgstr ""
+msgstr "Dydd Sadwrn y Cyfrifiad, 20 Mawrth"
 
-#: app/templates/support_centre_list_of_centres.html:130
+#: app/templates/support_centre_list_of_centres.html:132
 msgid "Census Day, 21 March"
-msgstr ""
+msgstr "Diwrnod y Cyfrifiad, 21 Mawrth"
 
-#: app/templates/support_centre_list_of_centres.html:139
+#: app/templates/support_centre_list_of_centres.html:141
 msgid "Good Friday, 2 April"
-msgstr ""
+msgstr "Dydd Gwener y Groglith, 2 Ebrill"
 
-#: app/templates/support_centre_list_of_centres.html:148
+#: app/templates/support_centre_list_of_centres.html:150
 msgid "Easter Monday, 5 April"
-msgstr ""
+msgstr "Dydd Llun y Pasg, 5 Ebrill"
 
-#: app/templates/support_centre_list_of_centres.html:157
+#: app/templates/support_centre_list_of_centres.html:159
 msgid "May Bank Holiday, 3 May"
-msgstr ""
+msgstr "Gŵyl Banc Calan Mai, 3 Mai"
 
-#: app/templates/support_centre_list_of_centres.html:168
+#: app/templates/support_centre_list_of_centres.html:170
 msgid "Centre facilities"
-msgstr ""
+msgstr "Cyfleusterau'r canolfannau"
 
-#: app/templates/support_centre_list_of_centres.html:172
+#: app/templates/support_centre_list_of_centres.html:174
 msgid "Car park, including disabled parking"
 msgstr ""
 
-#: app/templates/support_centre_list_of_centres.html:174
+#: app/templates/support_centre_list_of_centres.html:176
 msgid "Car park"
-msgstr ""
+msgstr "Maes parcio"
 
-#: app/templates/support_centre_list_of_centres.html:176
+#: app/templates/support_centre_list_of_centres.html:178
 msgid "Disabled parking"
 msgstr ""
 
-#: app/templates/support_centre_list_of_centres.html:178
+#: app/templates/support_centre_list_of_centres.html:180
 msgid "Level access into building entrance"
```

Comment:
Is this one of the messages that has been marked as waiting to be translated?

Reply:
Yes - for the 2nd time we weren't asked they just went of the pictures, so missed some of the accessibility marker text. I have now provided a list of the outstanding translations